spec/src/main/asciidoc/java-api.adoc

```diff
@@ -102,7 +161,8 @@ public class MyCheck implements HealthCheck {
 }
 ```
 
-As health check procedures are CDI bean they can also be defined with CDI producers:
+As health check procedures are CDI bean.
```

Comment:
This looks like an incomplete sentence. The original version just missed a comma :-)